addons/crm_mail_client_extension/models/crm_lead.py

```diff
@@ -3,11 +3,16 @@
 
 from odoo import api, models
 
+
 class Lead(models.Model):
     _inherit = 'crm.lead'
 
     @api.model
     def _form_view_auto_fill(self):
+        """
+            deprecated method, not needed for newer versions of the mail plugin but necessary
+            for supporting older versions
```

Comment:
Could be nice to say it's deprecated as of [VERSION].
Here you'd have saas-14.3 for example.